### PR TITLE
Stream options.

### PR DESCRIPTION
**Pull request by Narsil**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by andrewrreed** [comment]
Thanks for the quick turnaround on this! This PR closes #2531

File: nix/client.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = ../clients/python;
+
+  pyproject = true;
+
+  build-system = [ poetry-core ];
+
+  nativeBuildInputs = [ ];
+
+  pythonRemoveDeps = [ ];
```
Comment (danieldk) [inline nix/client.nix:19]: Can just be removed.

File: router/src/server.rs [hunk 2]
```diff
@@ -590,7 +590,7 @@ async fn generate_stream_internal(
                                         let event = on_message_callback(stream_token);
                                         yield Ok(event);
                                     }
-                                    // Yield event for last token and compute timings
+                                    // Yield event for lat token and compute timings
```
Comment (danieldk) [inline router/src/server.rs:593]: ???

**Review by danieldk — APPROVED** [review]
Looks good, two small nits.

**Pull request merged**